src/pages/landing/deriv-go/_deriv-go-profits.js

```diff
@@ -0,0 +1,111 @@
+import React from 'react'
+import styled from 'styled-components'
+import PropTypes from 'prop-types'
+import { graphql, useStaticQuery } from 'gatsby'
+import device from 'themes/device'
+import { Container, Flex, SectionContainer } from 'components/containers'
+import { Header, Text, QueryImage } from 'components/elements'
+
+const StyledSection = styled(SectionContainer)`
+    background: rgba(243, 246, 250, 1);
+    padding: 8.9rem 0 8rem 0;
+    @media ${device.tabletL} {
+        padding: 40px 16px;
+    }
+`
+const StyledContainer = styled(Container)`
+    flex-direction: column;
+    max-width: 996px;
+    @media ${device.tabletL} {
+        max-width: 360px;
+    }
+`
+const Content = styled(Flex)`
+    width: 100%;
+    max-width: 38.4rem;
+    display: flex;
+    flex-direction: column;
+    margin-right: ${(props) => props.margin_right};
+
+    ${Text} {
+        font-size: 2.4rem;
+        margin-top: 0.8rem;
+    }
+    @media ${device.tabletL} {
+        margin: 0 auto;
+    }
+`
+
+const ImageWrapper = styled.div`
+    width: 100%;
+    margin-right: ${(props) => props.margin_right};
+    @media ${device.tabletL} {
+        margin: 2rem auto;
+    }
+`
```

Comment:
`QueryImage` already has a wrapper around GatsbyImage. You can give it a className to target it for styling.